flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/codegen/calls/ScalarFunctions.scala

```diff
@@ -143,15 +143,30 @@ object ScalarFunctions {
     new MultiTypeMethodCallGen(BuiltInMethods.ABS))
 
   addSqlFunction(
+    ABS,
+    Seq(BIG_DEC_TYPE_INFO),
+    new MultiTypeMethodCallGen(BuiltInMethods.ABS_DEC))
+
+  addSqlFunction(
```

Comment:
Just wondering, why are there no functions for `FLOAT`? Are they handled by the `DOUBLE` functions?